src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -4,6 +4,7 @@
 
 use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
+use RuntimeException;
```

Comment:
@GrahamCampbell It looks like a good candidate for StyleCI, e.g. for Python [isort](https://pypi.python.org/pypi/isort), [flake8-import-order](https://github.com/PyCQA/flake8-import-order), etc.